doc/contributing/advocacy-ambasador-program.md

```diff
@@ -0,0 +1,90 @@
+# Advocacy Ambassador program
+
+Node.js aims to support and encourage developers and developer
+advocates to promote Node.js as a project and to help communicate
+key messages to their constituencies.
+
+The ambassador program does that by:
+
+* Maintaining an up-to-date set of messages that the Node.js maintainers would like ambassadors
+  to promote. These will be documented in the nodejs/ambassadors repository.
+* Maintaining an up-to-date set of topics on which additional content
+  would benefit the community. These will be documented in the
+  nodejs/ambassadors repository.
+* Providing a space for collaborators to share new topics/features they
+  are working on when they would be happy for ambassadors to help share and get
+  early feedback.
+* Helping ambassadors promote content that aligns with the
+  messages and topics defined.
+* Advocating for ambassadors to be part of the OpenJS speakers bureau, even if the
+  ambassador is not otherwise an active member of the project itself.
+
+## Ambassadors nominations
+
+Any member of the Node.js organization may nominate an ambassador.
+The Technical Steering Committee (TSC) will review/approve ambassadors.
+
+In the initial phase of the program:
+
+* There will be a maximum of 4 ambassadors.
+* The term will be for the calendar year. After a year, ambassadors automatically
+  drop out of the ambassador list and must be re-nominated.
+* Ambassadors will be documented on the nodejs/ambassadors `README.md`.
+
+The TSC may remove an ambassador for any reason,
+including promoting content misaligned with the project.
+
+Members of the Node.js organization can nominate ambassadors by opening
+a discussion issue with the nomination in
+[nodejs/collaborators](https://github.com/nodejs/collaborators/)
+titled `Nomination X to be an Ambassador`, where X is the name of the person
+being nominated.
+
+## Reviewing content
+
+Ambassadors are asked to share links to new content before asking
```

Comment:
nit: `Ambassadors will be asked` opposed to `Ambassadors are asked`

Reply:
I don't think we will ask them later, they are asked in general, so Ambassadors are asked I think is better.

Reply:
I'm not sure, IMO it sounds a bit off, but I'm happy either way